Makefile

```diff
@@ -48,7 +46,7 @@ endif
 
 # Set the Operator SDK version to use. By default, what is installed on the system is used.
 # This is useful for CI or a project to utilize a specific version of the operator-sdk toolkit.
-OPERATOR_SDK_VERSION ?= v1.34.1
+OPERATOR_SDK_VERSION ?= v1.33.0
```

Comment:
Why a lower version?

Reply:
There seems to be an issue with 1.34.1 on running the bundle target (`make bundle`) that produces incomplete artifacts:
https://github.com/operator-framework/operator-sdk/issues/6698
I reverted to use the previously available version (1.33.0) to avoid the issue mentioned.